ci-operator/step-registry/openshift-qe/concurrent-builds/openshift-qe-concurrent-builds-ref.yaml

```diff
@@ -0,0 +1,30 @@
+ref:
+  as: openshift-qe-concurrent-builds
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  commands: openshift-qe-concurrent-builds-commands.sh
+  credentials:
+  - namespace: test-credentials # this entry injects the custom credential
+    name: ocp-qe-perfscale-es
+    mount_path: /secret
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  env:
+  - name: PODS_PER_NODE
```

Comment:
this variable isn't supported with the concurrent builds workload. Need to specify these 2: https://github.com/cloud-bulldozer/e2e-benchmarking/blob/f7641e36cc07271205be4cc7e15b0c2097a1f7ac/workloads/kube-burner/env.sh#L63-L64